Tribler/Core/Modules/cache.py

```diff
@@ -0,0 +1,54 @@
+import codecs
+import os
+
+
+class SimpleCache(object):
+    """
+    This is a cache for recording the keys that we have seen before.
+    """
+    def __init__(self, file_path):
+        self._file_path = file_path
+
+        self._cache_list = list()
+        self._initial_cache_size = 0
+
+    def add(self, key):
+        if not self.has(key):
+            self._cache_list.append(key)
+
+    def has(self, key):
+        return key in self._cache_list
+
+    def load(self):
```

Comment:
maybe better to use json or pickle instead of your own parser for simplicity sake?